subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/codenarc/CodeNarcCross_VersionIntegrationTest.groovy

```diff
@@ -23,7 +23,7 @@ import org.gradle.util.internal.VersionNumber
 
 import static org.hamcrest.CoreMatchers.startsWith
 
-class CodeNarcCrossVersionIntegrationTest extends AbstractIntegrationSpec implements CodeNarcTestFixture {
```

Comment:
I didn't even think about the name causing problems.  Let's just call it `CodeNarcGroovyVersionInegrationTest` to avoid any misunderstanding about what this does.